tests/plugin_tests.cpp

```diff
@@ -0,0 +1,29 @@
+#include <gtest/gtest.h>
+#include "plugin.h"  // You'll need to create this header file
```

Comment:
Should this be commented out or even pushed? I am not sure where we are going to implement testing or if we are going to change the testing format or not? I am very confused here and we should confirm this with @harshitha-thota

Reply:
I have the same confusion with the CMAKE File up top

Reply:
Yes I have a new file this should be removed.